starters/vue3-apollo-quasar/src/layouts/MainLayout.vue

```diff
@@ -36,8 +33,15 @@ useMeta({
 </script>
 
 <style lang="scss">
+.page-header {
+  width: 100%;
+  margin: 20px auto;
```

Comment:
avoid to use absolute values. especially for sizes and positioning properies

Reply:
Guess this is fixed now.Thanks